packages/auto-label/src/auto-label.ts

```diff
@@ -34,6 +34,7 @@ import {
 } from '@google-automations/bot-config-utils';
 import {syncLabels} from '@google-automations/label-utils';
 
+const API_NA_LABEL = 'api: N/A';
```

Comment:
Please add a comment describing that N/A means 'not applicable', and describe why users would add it.